cli/api.py

```diff
@@ -50,8 +50,9 @@ def check_auth(id_token: str) -> Optional[str]:
     """Check if an id_token is valid by making a request to the base API URL."""
     response = requests.get(_url('/'), headers=_with_auth(id_token=id_token))
 
-    # 401 Unauthorized, so token is invalid
-    if response.status_code == 401:
+    # 401 Unauthorized: token is invalid
+    # 412 Precondition Failed: CLI is out-of-date
+    if response.status_code == 401 or response.status_code == 412:
```

Comment:
Maybe all of 4xx?

Reply:
and even think on displaying some clues from 5xx too? Like 504/503/502 - may be try later, 500 - contact us?